enterprise/backend/src/metabase_enterprise/audit_db.clj

```diff
@@ -75,21 +75,41 @@
     (if-let [audit-db (t2/select-one :model/Database {:where [:= :is_audit true]})]
       (do (log/info "Beginning Audit DB Sync...")
           (log/with-no-logs (sync-metadata/sync-db-metadata! audit-db))
-          (log/info "Audit DB Sync Complete."))
+          (log/info "Audit DB Sync Complete.")
+          ;; We need to move back to a schema that matches the serialized data
+          (when (= :h2 mdb.env/db-type)
+            (t2/update! :metabase_database (:id audit-db) {:engine "postgres"})
```

Comment:
```suggestion
            (t2/update! (t2/table-name :model/Database) (:id audit-db) {:engine "postgres"})
```

One of these days we're finally going to rename these tables to something more sensible, and we don't want to have to go update 1000 LoC in the app code